Shared/Extensions/URLExtensions.swift

```diff
@@ -153,8 +153,9 @@ extension URL {
      * Returns the second level domain (SLD) of a url. It removes any subdomain/TLD
      *
      * E.g., https://m.foo.com/bar/baz?noo=abc#123  => foo
+     *       https://accounts.foo.com/bar/baz?noo=abc#123  => accounts.foo
      **/
-    public var hostSLD: String {
+    public var shortDisplayString: String {
```

Comment:
More descriptive naming I prefer, but this is specialized so ok with me. 
My fantasy is that when we push Shared to a dedicated repo, we get collaboration from other users as to how to improve the naming conventions in this file to avoid ambiguity. I forgot to ask Santa for this.